util/src/entities/BaseClass.ts

```diff
@@ -1,63 +1,10 @@
 import "reflect-metadata";
-import { BaseEntity, EntityMetadata, FindConditions, ObjectIdColumn, PrimaryColumn } from "typeorm";
+import { BaseEntity, EntityMetadata, ObjectIdColumn, PrimaryColumn, FindOptionsWhere, Generated, SaveOptions } from "typeorm";
 import { Snowflake } from "../util/Snowflake";
-import "missing-native-js-functions";
 
 export class BaseClassWithoutId extends BaseEntity {
-	constructor(props?: any) {
+	constructor() {
 		super();
-		this.assign(props);
-	}
```

Comment:
I'd like if you added back this constructor and used the ormutils mergedeep func + used it instead of `mergeDeep(new Entity(), { props })`, it just makes more sense + removes the need for importing ormutils literally everywhere